### ISSUE TITLE
Dev/Preview versions will not Consistantly start and run

### ISSUE THREAD

**Issue by jpparametrics** [files-community/files-uwp#1402]
"Describe the bug"

Uninstalled the apps yesterday, installed the apps, Preview and Dev, work fine for a while. The Files will open the "Splsh Screen" so to speak and then exits. Uninstalled both versions this morning. Fired up VS and Files would not run stating that it needed to be deployed first which it already was. So, I downloaded the src again <0.11.3> and extracted the files and opened the solution form that location on my external drive. Worked fine this morning for awhile and then once again it stopped working altogether. Showing the "Splash Screen" and then exiting. I just downloaded the SRC again and am going to do the same steps as above WITHOUT the preview being installed.

OS-Ver: [Windows 10 Pro Build 20161.rs_prerelease.200627-1754]
AppVer: [v0.11.3]
Email:    [jonnypjohnston@gmail.com]

**Additional context**
Add any other context about the problem here. Does this problem occur again after restarting the app?

**Comment by jpparametrics** [comment]
***UPDATE***

I installed the Preview version first, did not start it up. I then started the Dev version through VS and the Dev version is running.

When I start the Preview version it gets stuck at the attached screenshot. There is no user on this box as it shows in the screenshot either. That is on a box that a friend is using at his house.
![2020-07-12-Preview](https://user-images.githubusercontent.com/3137489/87250022-629aae00-c430-11ea-9cfb-e22262aad815.png)
